ports/littlegptracker/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "littlegptracker.zip",
+  "items": [
+    "LittleGPTracker.sh",
+    "littlegptracker"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "LittleGPTracker",
+    "porter": [
+      "ohol-vitaliy"
```

Comment:
Double check for a typo

Reply:
> Sorry, don't see typos in this file

Thought it might have been supposed to be `ohol-vitality`. If it's correct as is, np.